engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -317,9 +308,9 @@ private void generateSkyBand(FBO skyBand) {
         }
 
         skyBand.bind();
-        graphicState.setRenderBufferMask(skyBand, true, false, false);
+        setRenderBufferMask(skyBand, true, false, false);
 
-        setViewportTo(skyBand.dimensions());
+        setViewportToSizeOf(skyBand.dimensions());
```

Comment:
Hmmmm... I do not see the rationale of leaving unused methods. There might be an argument for methods used elsewhere, but if you are moving a method from one place to another I do not see an argument in keeping the old version, especially if it's not used anywhere else.